crates/turborepo-repository/src/package_graph/builder.rs

```diff
@@ -690,6 +690,23 @@ impl PackageInfo {
     }
 }
 
+#[allow(dead_code)]
+pub struct MockDiscovery;
```

Comment:
We should definitely make sure this is only usable in tests. If we expect this to be shared more widely, it should move out of the package graph builder tests.
```suggestion
#[allow(dead_code)]
#[cfg(test)]
pub struct MockDiscovery;
```